MSAL/src/native_auth/public/state_machine/delegate/ResetPasswordDelegates.swift

```diff
@@ -88,5 +88,5 @@ public protocol ResetPasswordRequiredDelegate {
     @MainActor func onResetPasswordRequiredError(error: PasswordRequiredError, newState: ResetPasswordRequiredState?)
 
     /// Notifies the delegate that the reset password operation completed successfully.
-    @MainActor func onResetPasswordCompleted()
+    @MainActor @objc optional func onResetPasswordCompleted()
```

Comment:
can you please add a note in the comment that says something like (we already have these kinds of notes):
`    /// - Note: If a flow requires this optional method and it is not implemented, then ``onResetPasswordRequiredError(error)`` will be called. ` 
I'd say to use a generic note whenever possible